### PR TITLE
[MODULAR] Medigun/Cellgun Code Cleanup

### PR DESCRIPTION
**Pull request by softcerv**
## About The Pull Request
Cleans up a good chunk of the code within the files for Cellguns and Mediguns. Adds in spaces to increase readability, removes an instance of a single letter variable, changes variable comments to work with autodoc.  

## How This Contributes To The Skyrat Roleplay Experience
More readable code is HRP ;)

## Changelog

:cl:
code: cleans up cellgun code
code: cleans up medigun code
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/modular_weapons/code/cellguns.dm [hunk 1]
```diff
@@ -45,26 +56,32 @@
 
 	var/ratio = get_charge_ratio()
 	ratio = get_charge_ratio()
+
 	if(ratio == 0 && display_empty)
```
Comment (Zonespace27) [inline modular_skyrat/modules/modular_weapons/code/cellguns.dm]:
```suggestion
	if(!ratio && display_empty)
```

File: modular_skyrat/modules/modular_weapons/code/cellguns.dm [hunk 2]
```diff
@@ -45,26 +56,32 @@
...
 	charge_overlay.color = shot.select_color
+
 	for(var/i = ratio, i >= 1, i--)
 		charge_overlay.pixel_x = ammo_x_offset * (i - 1)
 		charge_overlay.pixel_y = ammo_y_offset * (i - 1)
 		. += new /mutable_appearance(charge_overlay)
 
 /obj/item/gun/energy/cell_loaded/AltClick(mob/user, modifiers)
-	if(cellcount >= 1) //Is there a cell inside?
+	if(cellcount >= 1) //Checks to see if there is a cell inside of the gun, before removal.
```
Comment (Zonespace27) [inline modular_skyrat/modules/modular_weapons/code/cellguns.dm]:
```suggestion
	if(cellcount) //Checks to see if there is a cell inside of the gun, before removal.
```

File: modular_skyrat/modules/modular_weapons/code/cellguns.dm [hunk 3]
```diff
@@ -45,26 +56,32 @@
...
 		return
 
 	var/mutable_appearance/charge_overlay = mutable_appearance(icon, overlay_icon_state)
+
 	if(!shot.select_color)
 		return
+
 	charge_overlay.color = shot.select_color
+
 	for(var/i = ratio, i >= 1, i--)
```
Comment (Zonespace27) [inline modular_skyrat/modules/modular_weapons/code/cellguns.dm]: use `for(var/i in numstart to numend)`

File: modular_skyrat/modules/modular_weapons/code/medcells.dm [hunk 4]
```diff
@@ -22,78 +24,100 @@
...
 	else
 		return TRUE
-//Checks for non-medicine reagents in the bloodstream
+
+/// Checks for non-medicine reagents in the bloodstream, used for the toxin medicell.
 /obj/projectile/energy/medical/proc/checkReagents(mob/living/target)
 	var/non_medicine_chems = 0 //Keeps track of how many chemicals in the bloodstream aren't medicine.
+
 	for(var/reagent in target.reagents.reagent_list)
 		if(!istype(reagent, /datum/reagent/medicine))
```
Comment (Zonespace27) [inline modular_skyrat/modules/modular_weapons/code/medcells.dm]: early continue

File: modular_skyrat/modules/modular_weapons/code/cellguns.dm [hunk 5]
```diff
@@ -1,29 +1,39 @@
...
 			. += span_notice("There is \a [cell] loaded in the chamber.") //Shows what cells are currently inside of the gun
 
-/obj/item/gun/energy/cell_loaded/attackby(obj/item/weaponcell/C, mob/user) //Inserts a cell.
-	if(is_type_in_list(C, allowed_cells))//Checks to see if the gun has the capacity based on allowed_types to load a cell.
-		if(cellcount >= maxcells) //Are there too many cells inside of the gun?
+/// Handles insertion of weapon cells
+/obj/item/gun/energy/cell_loaded/attackby(obj/item/weaponcell/used_cell, mob/user)
+	if(is_type_in_list(used_cell, allowed_cells)) // Checks allowed_cells to see if the gun is able to load the cells.
+		if(cellcount >= maxcells) //Prevents the user from loading any cells past the maximum cell allowance
 			to_chat(user, span_notice("The [src] is full, take a cell out to make room"))
```
Comment (Zonespace27) [inline modular_skyrat/modules/modular_weapons/code/cellguns.dm]:
```suggestion
			to_chat(user, span_notice("[src] is full, take a cell out to make room."))
```

File: modular_skyrat/modules/modular_weapons/code/cellguns.dm [hunk 6]
```diff
@@ -36,6 +46,7 @@
 	. = ..()
 	var/overlay_icon_state = "[icon_state]"
```
Comment (Zonespace27) [inline modular_skyrat/modules/modular_weapons/code/cellguns.dm]:
```suggestion
	var/overlay_icon_state = icon_state
```

File: modular_skyrat/modules/modular_weapons/code/cellguns.dm (hunk 2 above)
Comment (jjpark-kb) [inline modular_skyrat/modules/modular_weapons/code/cellguns.dm]: this should probably be an early return

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**